tests/_benderjs/ckeditor/static/tools.js

```diff
@@ -59,6 +59,19 @@
 		} );
 
 	bender.tools = {
+		/**
+		 * Creates an array from an object.
+		 *
+		 * @param  {Object} obj
+		 * @return {Array} object values.
+		 */
+		objToArray: function( obj ) {
```

Comment:
This change does not seem to be directly connected with the fix. Why is it present here?